packages/jest-haste-map/src/index.ts

```diff
@@ -746,11 +737,11 @@ export default class HasteMap extends EventEmitter {
   /**
    * Creates workers or parses files and extracts metadata in-process.
    */
-  private _getWorker(options?: {
-    forceInBand: boolean;
-  }): JestWorkerFarm<HasteWorker> | HasteWorker {
+  private _getWorker(
+    options = {forceInBand: false},
```

Comment:
`false`, not `boolean`?

Reply:
The idea was to have default value. `options` object always has some value, so there is no need for `options && ` check later.

Reply:
oh, right, misread